cmd/sealer/cmd/image/load.go

```diff
@@ -58,7 +62,7 @@ func NewLoadCmd() *cobra.Command {
 	flags := loadCmd.Flags()
 	flags.StringVarP(&loadOpts.Input, "input", "i", "", "Load image from file")
 	flags.BoolVarP(&loadOpts.Quiet, "quiet", "q", false, "Suppress the output")
-	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "", "set temporary directory when load image. if not set, use system`s temporary directory")
+	flags.StringVar(&loadOpts.TmpDir, "tmp-dir", "d", "Set temporary directory when load image. if not set, use system temporary directory(`/var/tmp/`)")
```

Comment:
`-d` will conflict with debug mode?
`set` should be lowercase

Reply:
I'll go back and finish it off. `-d, --debug`